### ISSUE TITLE
Function gets called twice when input bound to multiple event handlers

### ISSUE THREAD

**Issue by vicinting** [glortho/react-keydown#39]
**Functions**

```
  @keydown('shift+enter', 'ctrl+enter', 'alt+enter')
  handleInputKeydown(e) {    
    console.log('next line');
  }
  @keydown('enter')
  handleSubmit(e) {
    console.log('submit');
  }
```

**Text Area**

```
<textarea
                  name="content"
                  placeholder={placeholderText}
                  value={this.state.content}
                  onChange={this.handleInputChange}
                  onKeyDown={(e) => { this.handleInputKeydown(e); this.handleSubmit(e) }} />
```

Did the above instead of just one function since I couldn't find a way to discern between a regular Enter key and Shift+Enter. The event prop in the function just gives you one key (last pressed I assume), so I separated them into 2 functions.

It's working fine, however, the function seems to be called twice. When I hit enter, I see 'submit' in the console twice.

**Comment by glortho** [comment]
Same issue as #38 . I'll see if I can fix this in the next day or two.

**Comment by glortho** [comment]
This should be fixed as of 1.6.9. Test it and let me know if not!
